export-ingester/export_ingester/utils.py

```diff
@@ -5,8 +5,8 @@ def now() -> datetime:
     return datetime.now(tz=UTC)
 
 
-def get_range(date_: date) -> tuple[datetime, datetime]:
-    next_day = date_ + timedelta(days=1)
+def get_range(date_: date, time_range: int) -> tuple[datetime, datetime]:
```

Comment:
I think this complicates things a bit - see above suggestion re `get_date_start`